src/validators/ipv4-range-mask

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I believe I should add range option to `ipaddrcheck`. I'll look into it, if it takes a while, we can merge this first, then replace the validator part.